### ISSUE TITLE
Triode's Switch Player plugin function

### ISSUE THREAD

**Issue by kaaholst** [kaaholst/android-squeezer#82]
```
This is a feature I use a lot for moving the playback from room to room and is pretty
much the remaining reason why I switch back to the logitech controller app on a daily
basis.
Would love to see this listed in the squeezer home menu.
Thanks.

```

Original issue reported on code.google.com by `drdocmatt` on 2014-02-15 11:54:30

**Comment by kaaholst** [comment]
https://github.com/nikclayton/android-squeezer/wiki/Apps-support-in-Squeezer


Original issue reported on github.com/nikclayton/android-squeezer by nikclayton on 2015-06-01T11:35:57Z

**Comment by kaaholst** [comment]
Fixed in 2.0.0

Original issue reported on github.com/nikclayton/android-squeezer by kaaholst on 2020-03-07T19:07:53Z
